contracts/tests/SimplePriceFeed.sol

```diff
@@ -0,0 +1,80 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
```

Comment:
SimplePriceFeed is used for stableSwapModule. I forgot to mention that we either have to get rid of priceLife at all so US+'s price will always be 1, or just make priceLife that is equal or longer than 365 days to keep US+'s price 1 for a long long time.